### ISSUE TITLE
build: test-docker is always building master

### ISSUE THREAD

**Issue by ingwinlu** [ElektraInitiative/libelektra#1913]

**Comment by ingwinlu** [comment]
I think so yes.

Take this log for `test-docker` from #1910 for example:
```sh
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/ElektraInitiative/libelektra.git # timeout=10
Fetching upstream changes from https://github.com/ElektraInitiative/libelektra.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/ElektraInitiative/libelektra.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision b1a8e1bed9c3aa4dc6beeae01b43251eedc5cbd2 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f b1a8e1bed9c3aa4dc6beeae01b43251eedc5cbd2
 > git branch -a -v --no-abbrev # timeout=10
 > git branch -D master # timeout=10
 > git checkout -b master b1a8e1bed9c3aa4dc6beeae01b43251eedc5cbd2
Commit message: "doc: add missing [type]"
 > git rev-list --no-walk b1a8e1bed9c3aa4dc6beeae01b43251eedc5cbd2 # timeout=10
```

And compare this with one that is setup correctly (`elektra-gcc`):
``` sh
Cloning repository git://github.com/ElektraInitiative/libelektra.git
 > git init /home/jenkins/workspace/elektra-gcc # timeout=10
Using reference repository: /home/jenkins/git_mirrors/libelektra
Fetching upstream changes from git://github.com/ElektraInitiative/libelektra.git
 > git --version # timeout=10
 > git fetch --tags --progress git://github.com/ElektraInitiative/libelektra.git +refs/heads/*:refs/remotes/origin/* --depth=1
 > git config remote.origin.url git://github.com/ElektraInitiative/libelektra.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url git://github.com/ElektraInitiative/libelektra.git # timeout=10
Fetching upstream changes from git://github.com/ElektraInitiative/libelektra.git
 > git fetch --tags --progress git://github.com/ElektraInitiative/libelektra.git +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/* --depth=1
 > git rev-parse refs/remotes/origin/pr/1909/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/1909/merge^{commit} # timeout=10
Checking out Revision a0375f37eecf4303f58aa1964ccadf40e9e94674 (refs/remotes/origin/pr/1909/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f a0375f37eecf4303f58aa1964ccadf40e9e94674
Commit message: "Merge 356317a6ab811f524ebaa1dbf40f19477b966fc1 into b1a8e1bed9c3aa4dc6beeae01b43251eedc5cbd2"
```

**Comment by ingwinlu** [comment]
I did disable `docker-test` for now 
cc #160

**Comment by markus2330** [comment]
Thank you!

Once your tests cover everything from docker-test, docker-test should be deleted.

**Comment by ingwinlu** [comment]
Can be closed once #1866 is merged.

**Comment by markus2330** [comment]
Thank you for the notice! Simply write closes #1913 within #1866.

**Comment by ingwinlu** [comment]
resolved via #1866
